test/boost/s3_test.cc

```diff
@@ -35,15 +35,15 @@
 // permissive enough policy and then run the test with env set respectively
 // E.g. like this
 //
-//   export PROXY_S3_SERVER_HOST=s3.us-east-2.amazonaws.com
+//   export S3_SERVER_ADDRESS_FOR_TEST=s3.us-east-2.amazonaws.com
```

Comment:
the title of the commit message could be confusing without more context, how about using :


> test/boost/s3_test: add error injection scenarios to existing test suite

Reply:
done